config/settings/production.py

```diff
@@ -33,14 +33,7 @@
 # Make sure SecurityMiddleware is listed first
 MIDDLEWARE_CLASSES = SECURITY_MIDDLEWARE + MIDDLEWARE_CLASSES
 
-# set this to 60 seconds and then to 518400 when you can prove it works
-SECURE_HSTS_SECONDS = env.int('DJANGO_SECURE_HSTS_SECONDS', default=60)
-SECURE_HSTS_INCLUDE_SUBDOMAINS = True
-SECURE_CONTENT_TYPE_NOSNIFF = True
-SECURE_BROWSER_XSS_FILTER = True
-SESSION_COOKIE_SECURE = True
```

Comment:
Well true, `SESSION_COOKIE_SECURE` should probably not be removed, but moving other strictly header related out of Django allows them to be set and applied for all requests going through the web server.